app/src/main/java/com/example/habitsmasher/ListFragment.java

```diff
@@ -1,8 +1,9 @@
 package com.example.habitsmasher;
 
+import android.view.View;
```

Comment:
What's the point of this line?

Reply:
View is used on line 77 as a parameter within **initializeRecyclerView**, however, this is not anything newly added. On the left side, it was removed from line 5 and moved to line 3(here), which I believe probably happened when I opened the file and it was automatically moved around.